tests/test_post_process.py

```diff
@@ -212,3 +216,21 @@ def test_vaf(tmp_path):
         assert vaf[i] == approx(expected, rel=1e-9)
         assert lags == approx(w_lag, rel=1e-9)
         assert vaf[i] == approx(w_vaf, rel=1e-9)
+
+
+def test_vaf_by_symbols(tmp_path):
+    """Test vaf using element symbols."""
+    vaf_names = ("vaf-Na-by-indices.dat", "vaf-Na-by-element.dat")
+    vaf_filter = ((0, 2, 4, 6), ("Na",))
+
+    data = read(DATA_PATH / "NaCl-traj.xyz", index=":")
+    lags, vaf = post_process.compute_vaf(
+        data, atoms_filter=vaf_filter, filenames=[tmp_path / name for name in vaf_names]
+    )
+    expected = np.loadtxt(tmp_path / "vaf-Na-by-indices.dat")
+    actual = np.loadtxt(tmp_path / "vaf-Na-by-element.dat")
+
+    assert expected == approx(actual, rel=1e-9)
+
+    with pytest.raises(ValueError):
```

Comment:
```suggestion
    # Check invalid symbol
    with pytest.raises(ValueError):
```

This maybe ought to be its own test, but probably a comment is enough to give it some context